cupy/__init__.py

```diff
@@ -59,7 +59,6 @@ def is_available():
 from cupy import functional  # NOQA
 from cupy import indexing  # NOQA
 from cupy import io  # NOQA
-from cupy import linalg  # NOQA
```

Comment:
As in `numpy`, the `linalg` submodule should be available via `import cupy; cupy.linalg` or `from cupy import *; linalg`.

Reply:
I've fixed it. Thanks!